File: container_mgmt/lib/models/truck.dart
class Truck {
  final int truckId;
  final String truckName;

  Truck({required this.truckId, required this.truckName});

  factory Truck.fromJson(Map<String, dynamic> json) =>
      Truck(truckId: json['truckId'], truckName: json['truckName']);
}

File: con_mgmt_api/Models/Truck.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace ContainerManagementApi.Models;

[Table("Trucks")]
public class Truck
{
    [Key]
    public int TruckId { get; set; }

    [Required]
    [MaxLength(50)]
    public string TruckName { get; set; } = string.Empty;
}
